crates/libs/windows/src/Windows/Win32/Graphics/Direct3D12/mod.rs

```diff
@@ -25710,12 +25644,12 @@ impl ::core::default::Default for D3D12_RENDER_TARGET_VIEW_DESC_0 {
 #[repr(C)]
 #[doc = "*Required features: `\"Win32_Graphics_Direct3D12\"`*"]
 pub struct D3D12_RESOURCE_ALIASING_BARRIER {
-    pub pResourceBefore: ::core::option::Option<ID3D12Resource>,
-    pub pResourceAfter: ::core::option::Option<ID3D12Resource>,
+    pub pResourceBefore: ::windows::core::ManuallyDrop<ID3D12Resource>,
```

Comment:
Is it possible to store the equivalent of `None`  in a ManuallyDrop - so these show up as NULL when passed to the D3D12 runtime?

Reply:
Yes, `ManuallyDrop::none()` will give you that.

Reply:
I added this test to illustrate: https://github.com/microsoft/windows-rs/pull/2235/commits/d9be39dc2f3808d15ca32800545d1967908a31bc

Reply:
Cool.  Is this something specific about windows-rs interfaces?  I can't find any documentation for none on https://doc.rust-lang.org/std/mem/struct.ManuallyDrop.html.

Reply:
Yes. The `ManuallyDrop` provided by the standard library basically forces ownership, so the `windows` crate now provides a simple `ManuallyDrop` alternative for use in such cases reference semantics are desired. See here: 

https://github.com/microsoft/windows-rs/pull/2235/commits/18546d2361abeac538b1e86ea03599d86761ea5a#diff-594f4615fa893560c293b924a5380f92ad1c39e74b45a643b979c14c1a957c88

Reply:
Ah, I see. Any concerns about possible confusion to the same named struct in std::mem?